addons/website_event_track_live/views/event_track_templates_list.xml

```diff
@@ -1,19 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
-<template id="tracks_display_list" inherit_id="website_event_track.tracks_display_list">
+<template id="track_list_item" inherit_id="website_event_track.track_list_item">
     <!-- TRACK LIST: ADD REPLAY TAG FOR FINISHED TRACKS -->
-    <xpath expr="//div[@t-foreach='tracks']//div[hasclass('col-12')]//span[@t-elif='not track.is_track_done and not track.is_track_soon']" position="after">
+    <xpath expr="//div[hasclass('col-12')]//span[@t-elif='not track.is_track_done and not track.is_track_soon']" position="after">
```

Comment:
Seems like there is an issue with the design of the track displaying the replay thing, can you have a look ?

![Screenshot from 2025-01-23 13-02-00](https://github.com/user-attachments/assets/5c33935e-2289-4424-a068-37d2be444d30)

Reply:
Hello @jeh-odoo  the issue occurs because we're inserting an <a> inside another <a>, which breaks the layout as the browser restructures it into <a>Track card</a> <a>Replay</a>. Since the entire card is already clickable and the replay button redirects to the same talk, do we need to handle the click event separately for the replay button, or can we rely on the event loop and simply display the replay badge? :thinking:

Reply:
In that case, perhaps we can go for displaying the info like we do with "Live" and "Finished", I don't think we really need the replay tag to be clickable. Just check with the PO to be sure we don't miss some case :slightly_smiling_face:

Reply:
Agreed, I'll check with PO then.Thanks :)